app/server.go

```diff
@@ -1094,6 +1095,24 @@ func runLicenseExpirationCheckJob(a *App) {
 	}, time.Hour*24)
 }
 
+func runReportToAWSMeterJob(a *App) {
```

Comment:
This can be done receiving only the server, and instanciating it as part of the `app/server.go` `Server.RunJobs` method. This way you are coupling less this feature with the `App` layer, and only coupling it to the server layer.

Reply:
Sure - though at a certain point, as I was mentioning in a previous comment, might make sense to spend a bit of time and capture these guidelines so that you or somebody else from the server team don't have to repeat yourselves when people don't follow the recommended patterns.

Reply:
I going to raise that at the next server meeting, probably we should be capturing this kind of feedback from the pull requests and adding to some place (not only here, but almost as part of any PR). Thanks.